easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-foss-2015b.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Author: Adam Huffman
+# adam.huffman@crick.ac.uk
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'VCFtools'
+version = '0.1.14'
+
+homepage = "https://vcftools.github.io"
+description = """The aim of VCFtools is to provide 
+ easily accessible methods for working with complex 
+ genetic variation data in the form of VCF files."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+perl = 'Perl'
+perlver = '5.20.2'
+perlsuffix = '-bare'
+dependencies = [
+    (perl, perlver, perlsuffix),
```

Comment:
@boegel - this is what I was getting at in the earlier comment.

It's not clear to me what the policy is, if any, of when we need to be explicit about these suffixes and what the basis is for choosing them in the first place.

Maybe it's just because I'm used to RPM spec files.

In this case, I didn't want to add yet another large .eb for my small one (there's no non-bare recent Perl for this toolchain), but I suppose that is the cleanest approach.